CHANGELOG.md

```diff
@@ -12,6 +12,14 @@
 
 ---
 
+[//]: # (START/v1.4.0-beta01)
+# v1.4.0-beta01
```

Comment:
These two should match the `.VERSION` file, here's a quick code suggestion to address this:

```suggestion
[//]: # (START/v1.4.1-beta01)
# v1.4.1-beta01
```

Reply:
Nice catch!